libethereum/State.cpp

```diff
@@ -541,6 +541,7 @@ std::pair<ExecutionResult, TransactionReceipt> State::execute(EnvInfo const& _en
 	u256 const startGasUsed = _envInfo.gasUsed();
 	executeTransaction(e, _t, onOp);
 
+	res.changeLog = m_changeLog;
```

Comment:
I think it's essentially the same, let's call it change log.

We need it after transaction execution (`executeTransaction` call on line 542), but before `commit` which clears changelog.

The getter function would work if we save the copy before commit, or don't clear it.